server/services/deviceService.ts

```diff
@@ -21,10 +21,10 @@ export default class DeviceService {
 
   async findDeviceById(token: string, deviceId: string): Promise<Device> {
     try {
-      const response = await this.restClient(token).get<ApiResponse<'device', Device>>({
+      const response = await this.restClient(token).get<ApiResponse<'devices', Device[]>>({
```

Comment:
This is more a question for the API, but why is it returning a list for a request that should return 1 result?

Reply:
we are always returning lists from the API, they can be empty or have 1 or more elements.